packages/datatrak-web/src/AppProviders.tsx

```diff
@@ -11,6 +11,7 @@ import { SnackbarProvider } from 'notistack';
 import { theme } from './theme';
 import { Toast } from './components';
 import { errorToast } from './utils';
+import { CurrentUserContextProvider } from './api';
```

Comment:
I'm not really sure the best place to put this context file. I thought that it is related to the api data so it could go in the api folder